weave-js/src/components/Panel2/ChildPanelExportReport/computeReportSlateNode.ts

```diff
@@ -60,7 +68,18 @@ const getVarItemsForPath = (
 
   // First, extract the stack for a given panel.
   mapPanels(fullConfig, [], (config, stack) => {
-    if (config === targetConfig) {
+    if (
+      // mapPanels does not guarantee reference equality, so we need to use
+      // _.isEqual to compare the configs. This is at least the case for Group
+      // panels, which are cloned in mapPanels.
+      _.isEqual(config, targetConfig) ||
+      // Additionally, the target config might be a direct node itself, so we
+      // need to also check for equality with the input_node of the target
+      // config.
+      (isNodeOrVoidNode(targetConfig) &&
```

Comment:
Maybe we don't need this check other places, but just based on the names it feels like there should be some better single function that represents "isNodeOrVoidNode and !isVoidNode" - `isNonVoidNode`?.

Reply:
Added such a function and refactored in next commit